docs/source/workflows/sizing-calc.md

```diff
@@ -0,0 +1,330 @@
+```
+
+Run the Calculator using the remote endpoint:
+```
+aiq profile calc --config_file $CONFIG_FILE --output_dir $OUTPUT_DIR --concurrencies 1,2,4,8,16,32 --num_passes 2 --endpoint http://localhost:8000
+```
+The configuration file used for running the calculator only needs to specify the `eval` section. The `workflow` section is not used by the calculator when running with a remote endpoint.
+
+### Handling Failed Workflows
+Based on the test setup you may encounter failures as the concurrency value increases. When a workflow fails for an input the pass is stopped for that particular concurrency value. The pass is tagged with a `workflow_interrupted` flag in the JSON output. Such concurrencies, with a `workflow_interrupted` flag set to `true`, are not included in the GPU estimate. This information is indicated in the summary table via an `Alerts` column.
+
+**Sample output with alerts:**
+```
+Targets: LLM Latency ≤ 0.0s, Workflow Runtime ≤ 0.0s, Users = 0
+Test parameters: GPUs = 0
+Per concurrency results:
+Alerts: !W = Workflow interrupted
+| Alerts |   Concurrency |   p95 LLM Latency |   p95 WF Runtime |   Total Runtime |
+|--------|---------------|-------------------|------------------|-----------------|
+|        | 1             | 1.14981           | 4.03488          |         8.06977 |
+|        | 2             | 1.3591            | 4.71197          |         9.32298 |
+| !W     | 4             | 1.50682           | 5.67581          |         11.1683 |
+|        | 8             | 2.10668           | 7.90895          |         15.6193 |
+|        | 16            | 3.30196           | 12.677           |         25.3173 |
+|        | 32            | 6.57847           | 24.5307          |         43.9806 |
+```
+
+In this example, the workflow failed at concurrency level 4 (indicated by `!W` in the Alerts column). The time metrics for concurrency 4 are not included in the GPU estimate as they are not reliable and may skew the linear fit used to estimate the GPU count.
+
+## Estimate GPU Cluster Size
+Once the metrics are gathered, you can estimate the GPU cluster size using the `aiq profile calc` command in `offline_mode`.
+Sample command:
+```
+aiq profile calc --offline_mode --output_dir $OUTPUT_DIR --test_gpu_count 8 --target_workflow_runtime 10 --target_users 100
+```
+
+### Target and Test Parameters
+**Target Parameters**
+
+To estimate the GPU cluster size, you need to specify the target number of users and the target workflow runtime i.e. the maximum acceptable response time for the workflow.
+
+Optionally you can specify the target p95 LLM latency if the LLM latency is a defining factor for the workflow and if it is possible to measure the maximum acceptable LLM latency.
+- `target_users`: Target number of users to support.
+- `target_workflow_runtime`: Target p95 workflow runtime (seconds). Can be set to 0 to ignore.
+- `target_llm_latency`: Target p95 LLM latency (seconds). Can be set to 0 to ignore.
+
+**Test Parameters**
```

Comment:
I wonder if `--test_gpu_count` should be an input to the online mode that we store for users, rather than an input to the offline calculator?